test-harness/specs/validate-body-params/form-data-invalid-request.oas2.txt

```diff
@@ -27,7 +27,7 @@ paths:
 ====server====
 mock -p 4010
 ====command====
-curl -i -X POST http://localhost:4010/path -H "Content-Type: application/x-www-form-urlencoded" --data "status=ooopsie!"
+curl -i -X POST http://localhost:4010/path -H "Content-Type: application/x-www-form-urlencoded" --data "status=ooopsie%21"
```

Comment:
Just to make sure I understand this correctly — since `!` is a reserved character and `allowReserved` is false by default, you have effectively encoded it so that it passes the validation and goes through the following validation step, right?

Reply:
Exactly. And this is the place where I doubt in OpenApi 3 spec. I mean having `allowReserved=false` by default sounds reasonable. However, I think it will end with lots of people complaining that prism has a bug.

Reply:
It depends, usually the clients handle the encoding automatically and even curl has an option
`--data-urlencode`.

//cc @philsturgeon for awareness but I do not see this as a problem

Reply:
Updated to use `--data-urlencode` 👍